recipes/python/retail/Dockerfile

```diff
@@ -24,7 +30,6 @@ RUN /usr/bin/python3.5 -m pip install -U sklearn
 RUN /usr/bin/python3.5 -m pip install -U scipy
 RUN /usr/bin/python3.5 -m pip install -U fastparquet
 
-COPY dist/retail*.egg /application.egg
+RUN cp dist/retail*.egg /application.egg
```

Comment:
That is one big bug.

Reply:
IMO might be more readable to do the `cp` right after python install

Reply:
I agree. My only concern was if those previous pip installs need to be done after the setup.py, but I'm guessing not. So I can move and retest

Reply:
Maybe move the setup after the installs and before the cp

Reply:
Also the runtime has these dependencies any. Pretty sure these pip installs are unnecessary so I'll retry without them.